notebooks/Plot.ipynb

```diff
@@ -0,0 +1,467 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1045ad32-1580-4ff6-a422-a9d7e870bc21",
+   "metadata": {},
+   "source": [
+    "For the next example we create a new graph `G2` with 3 components: the first component contains the nodes 0-3, the second contains the nodes 4 and 5, and the third contains only node 6 (component of size 1)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9ccbf5e0-8ef5-45cb-b66a-4214651334d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G2=nk.Graph(7)\n",
+    "G2.addEdge(0,1) \n",
+    "G2.addEdge(1,2)\n",
+    "G2.addEdge(2,3)\n",
+    "G2.addEdge(4,5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e6abfb6-1482-4cbe-99e9-a69369db4020",
+   "metadata": {},
+   "source": [
+    "Plotting the `connectedComponentsSizes` of a graph `G` can be done the following way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8753a661-d298-4dd2-a454-84fe7cf412bb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nk.plot.connectedComponentsSizes(G2, relativeSizes=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d17750f0-4a08-43bd-b1fc-2eaed1d8ef59",
+   "metadata": {},
+   "source": [
+    "### NodeAttributes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57046d74-32f7-4298-b1cb-ad304954a052",
+   "metadata": {},
+   "source": [
+    "The `nodeAttributes` are useful in order to efficiently incorporate additional information in the graph data structure."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7c1d2ca5-71cb-4b37-91a1-5fa34f4d9ede",
+   "metadata": {},
+   "source": [
+    "Note: See [networkit.GraphTutorial](https://networkit.github.io/dev-docs/notebooks/GraphNotebook.html) in section `NodeAttributes` for a detailed description of how use them."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f0fb9b5-15b0-4d02-88bb-57666359757c",
+   "metadata": {},
+   "source": [
+    "For this example we create a new graph `G_attr` which contains additional graph attributes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e625b9e-d176-45e6-8949-dda910fd8c54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G_attr=nk.Graph(4)\n",
+    "G_attr.addEdge(0,1) \n",
+    "G_attr.addEdge(1,2) \n",
+    "G_attr.addEdge(2,3)\n",
+    "#attaching the attributes to the graph\n",
+    "attrFloat = G_attr.attachNodeAttribute(\"myFloat\", float)\n",
+    "attrStr = G_attr.attachNodeAttribute(\"myStr\", str)\n",
+    "#setting attribute values\n",
+    "attrFloat[0] = 0.5\n",
+    "attrFloat[1] = 1.5\n",
+    "attrStr[1] = \"tutorial\"\n",
+    "attrStr[2] = \"again\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b3fab65-4869-4947-91ac-6151aae40065",
+   "metadata": {},
+   "source": [
+    "Note: The `nk.plot.nodeAttributes` can only be called with one or two `nodeAttributes`. \n",
```

Comment:
Change according to the code review (only supported parameter formats: NodeAttribute and tuple of NodeAttributes).